### PR TITLE
Remove attr_writter for #number since it's already being overriden

### PR DESCRIPTION
**Pull request by tanmay3011**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://tanmay3011-use-attr-accessor.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://tanmay3011-use-attr-accessor.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: core/app/models/spree/credit_card.rb [hunk 1]
```diff
@@ -8,8 +8,8 @@ class CreditCard < Spree::Base
 
     after_save :ensure_one_default
 
+    attr_reader :number
     attr_accessor :encrypted_data,
-                    :number,
```
Comment (jasonfb) [inline core/app/models/spree/credit_card.rb]: how is the number supposed to be written to the credit card object without a write accessor?

Comment (tanmay3011) [inline core/app/models/spree/credit_card.rb]: We have a method in this same file at line 74 

def number=(num)
  @number = num.gsub(/[^0-9]/, '') rescue nil
end

which overrides the writter method provided by attr_accessor

Comment (jasonfb) [inline core/app/models/spree/credit_card.rb]: Got it.

**Pull request merged**